content/docs/use-cases/index.md

```diff
@@ -1,28 +1,28 @@
 # Use Cases
 
-We provide short articles on common ML workflow or data management scenarios
-that DVC can help with or improve. Our use cases are not written to be run
-end-to-end like tutorials. For more general, hands-on experience with DVC,
-please see our [Get Started](/doc/tutorials/get-started) instead.
+We provide short articles on common ML workflow and data science use cases that
```

Comment:
Is this a copy edit? I do agree about "and" but why did we change "scenarios"? If there's no SEO reason let's revert that please:

```suggestion
We provide short articles on common ML workflow and data science scenarios that
```

Reply:
Yes there is an SEO motivation here: the search term is "data science use cases".

Reply:
I see! Going fwd if you can make some notes in the PR file changes on terms each change is for, or a list of terms in the PR description at least, that would be helpful for reviews 😃

Reply:
👍 Definitely. That makes a lot of sense and I'll do that.

Reply:
Does it matter that probably users looking for "data science use cases" are not looking for DVC use cases? I don't want to assume what 1000s of people want, but it sounds like a basic data science question rather than anything to do with structuring DS projects (e.g. using DVC).

So maybe changes like this will bring more traffic but also up the bounce rate. We'll have to try and see, I guess!

Reply:
> Does it matter that probably users looking for "data science use cases" are not looking for DVC use cases?
It's true that the term is not a perfect match, but it is related to the primary subject area (data science). Most non-brand terms are going to be partially related but inexact, as searches for discovery are imprecise (because they don't know what DVC is yet). 

The search engine is trying to fill in the gaps, so we want to expand on terms that are showing interest within the correct subject area in order to meet them halfway. This article already has some impressions for "use cases", including ML and data science so that's the motivation for this change.

Reply:
OK, cool!